docs/release/release-note-0.3.0.md

```diff
@@ -0,0 +1,30 @@
+# Release Note 0.3.0
+
+## ONE-vscode
+
+### OneExplorerView
+
+- Support more commands
```

Comment:
I tried to find some sample release note and found that release note uses S + V sentenses.
e.g., `More commands are supported.` (Subject + Verb)  instead of `Support more commands` (imperative form starting with Verb)